portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java

```diff
@@ -85,6 +86,28 @@ public boolean equals(Object o) {
             Objects.equal(getAlbumId(), that.getAlbumId());
   }
 
+  // Assign this video to a different album. Used in cases where an album is too large and
+  // needs to be divided into smaller albums, the videos will each get reassigned to new
+  // albumnIds.
+  public void reassignToAlbum(String newAlbum){
+    this.albumId = newAlbum;
+  }
+
+  // remove all forbidden characters
+  public void cleanName(String forbiddenCharacters, char replacementCharacter, int maxLength) {
+    String name = getName().chars()
+        .mapToObj(c -> (char) c)
+        .map(c -> forbiddenCharacters.contains(Character.toString(c)) ? replacementCharacter : c)
+        .map(Object::toString)
+        .collect(Collectors.joining("")).trim();
+    if (maxLength <= 0) {
+      return;
+    }
```

Comment:
nit: 

this could have some performance impact. 
We are unnecessarily processing the name when the maxLength is <= 0. Why not push this condition to be the first stmt of the method. Otherwise, for thousands of files, the name would be processed.

Reply:
Hmm I copy pasted this from the photo model code, I think its not quite right, we still need to remove forbidden characters if max length is <=0, but in this code we actually never set the name after we remove the forbidden characters -- ill send a commit to fix this in both classes

Reply:
Ah I see, this is not a problem for the PhotosModel because we modify the title directly in the class instead of through the setName method.